pyproject.toml

```diff
@@ -91,8 +111,8 @@ legacy_tox_ini = """
     [testenv:coverage]
     description = get coverage report xml
     basepython = 3.9
-    extras = coveraging,pytesting,full
+    extras = coveraging,pytesting,full,dino
     commands =
-            coverage run -m pytest tests
+            coverage run -m pytest tests -m "notebooks"
```

Comment:
I don't think we want `-m "notebooks"` here because that only tests the notebooks. Just `coverage run -m pytest tests` which tests everything which is what we want for the coverage count.